.github/workflows/test.yml

```diff
@@ -1,12 +1,18 @@
 name: Test
 on:
   push:
+    tags:
+      - v*
+    branches:
+      - main
   pull_request:
-  schedule:
-  - cron: '0 9 * * 1'  # M H d m w (Mondays at 9:00)
```

Comment:
why remove this?

Reply:
Because, [unlike acceptance/smoke tests](https://github.com/iterative/terraform-provider-iterative/blob/4b385aa3937033af2ce1048596be7c41d5b01814/.github/workflows/smoke.yml#L6), results are deterministic.

Reply:
Would you expect unit tests to produce different results for the same commit provided we run them on different Mondays? 🤔

Reply:
well, I've had linter failures when `flake8` gets updated...

Reply:
Assuming that the linter preaks because of an update, we have another existential question: should we care?

Reply:
Anyhow, removing those lines is out of scope for this pull request.